.github/workflows/codeql.yaml

```diff
@@ -86,15 +88,15 @@ jobs:
     - name: Initialize CodeQL
       uses: github/codeql-action/init@v3
       with:
-        languages: ${{ matrix.language }}
-        build-mode: ${{ matrix.build-mode }}
+        languages: ${{ matrix.codeql.language }}
+        build-mode: ${{ matrix.codeql.build-mode }}
         queries: security-extended,security-and-quality
 
-    - if: matrix.build-mode == 'manual'
+    - if: matrix.codeql.build-mode == 'manual'
```

Comment:
Will this condition cause the Rust CodeQL run to be skipped because for Rust `build-mode` != `manual`?